pkg/tests/end_to_end_tests/trace_operation_calls_test.go

```diff
@@ -0,0 +1,378 @@
+package end_to_end_tests
```

Comment:
file license header please

Reply:
Is there a reason to add license headers to files? We have a repo-wide LICENSE file, so adding headers to each file is just extra clutter.

Reply:
I have added it, but I also question its utility. If we really need the license in each file, I would argue that we also need the license as a comment embedded in each database function's body.